shoop/campaigns/migrations/0006_campaign_effects.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import shoop.core.fields
+
+cached_values = []
+
+# FORWARDS
+def cache_values(apps, schema_editor):
+    print("\n--- Caching values ---")
```

Comment:
I don't know if it's good idea to add print statements to migrations. Maybe it would be better to just leave them out to not mess with anything that doesn't expect writes to stdout during migrations. (It could cause failures e.g. if stdout is closed.)